test/Interop/Cxx/stdlib/std-module-interface.swift

```diff
@@ -29,6 +29,4 @@
 // CHECK-STRING:   static func to_wstring(_ __val: Int32) -> std.__1.wstring
 // CHECK-STRING: }
 
-// CHECK-IOSFWD-NOT: static func to_string
-// CHECK-STRING-NOT: typealias string
```

Comment:
@zoecarver Looks like #39431 caused these to start getting imported. I removed them here in case that's fine, but just wanted to check if that's expected?

Reply:
I reverted that commit for the time being. I'm investigating why it's causing a few tests to fail in weird ways.

Reply:
Still reverting that patch while I investigate, but this test was disabled on main already, so you can probably rebase this now.